site/package.json

```diff
@@ -7,15 +7,15 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.0.1",
     "@testing-library/user-event": "^13.5.0",
+    "async": ">=2.6.4",
     "axios": "^0.26.1",
     "formik": "^2.2.9",
+    "nth-check": ">=2.0.1",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-scripts": "5.0.1",
     "styled-components": "^5.3.5",
-    "web-vitals": "^2.1.4",
-    "async": ">=2.6.4",
-    "nth-check": ">=2.0.1"
```

Comment:
Package changes are not required.

Reply:
Was this addressed, @AnnuCode?

Reply:
@leecalcote Any changes made to the `package.json` file have been reverted back to original state in the latest commit.